### PR TITLE
User Show Chart and UserSong.PlayedAt to DateTime

### PR DESCRIPTION
**Pull request by BrennanAyers**
This PR brings in the functionality of a chart being displayed on the Song show page for a specific User. As they follow through from their leaderboard on their Dashboard, they will see the beginning of historical stats on a specific song. The chart displays the Users PowerRanking over time on a line chart.
Each point is a date that the User listened to the song, and the PowerRanking that it received during that Activity. This is rendered by using the Chart.js library, piped through the `chartjs-ror` gem.

After creating this chart, and formatting the date to be part of the datapoint, I realized that our storing of UserSongs `played_at` attribute in Epoch time was not actually helpful. We had a team discussion to ensure that this would not be related to any other processes, and it was determined it was not.
To change the attribute to DateTime, we needed to change the database and then the SongSifter to follow in suit. Going forward, our rendering and comparison of times will be done with DateTime objects.

Additionally, our seed data was backwards for `album_art_url` and `spotify_url`. This pointed out the fact that the SongSifter had a mixed up array index in the building process for these two attributes. That has been resolved.

### CODE REVIEW

File: app/facades/song_show_facade.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+class SongShowFacade
+  def initialize(song, user)
+    @song = song
+    @user = user
+  end
+
+  def chart_data
```
[the remark this answers is not in the document]
Comment (BrennanAyers) [reply app/facades/song_show_facade.rb]: Does anybody care deeply about this? The way I see it, it's a Facade and it's providing data for a JS chart, it kind of needs to do a lot of things. I don't see a reasonable way to clean this up.

File: .hound.yml [hunk 2]
```diff
@@ -1,2 +1,2 @@
-rubocop:
```
Comment (alexander-mathieu) [inline .hound.yml]: 👍

File: app/assets/stylesheets/application.css [hunk 3]
```diff
@@ -15,11 +15,22 @@
...
-	padding: 14px 25px;
-	text-align: center; 
-	text-decoration: underline;
-	display: inline-block;
-								}
+  background-color: #f44336;
+  color: black;
+  border-radius: 1rem;
+  padding: 14px 25px;
+  text-align: center;
```
Comment (alexander-mathieu) [inline app/assets/stylesheets/application.css]: That's some nice CSS.

File: app/facades/song_show_facade.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (alexander-mathieu) [reply app/facades/song_show_facade.rb]: Sure don't.

File: db/seeds.rb [hunk 4]
```diff
@@ -1,5 +1,5 @@
 user1 = User.create!(strava_uid: 123456,
```
Comment (alexander-mathieu) [inline db/seeds.rb]: 🌱

**Pull request merged**